### DISCUSSION TITLE
Batch edited file Merge/Join feature?

### DISCUSSION THREAD

**Discussion by hkkane** [mifi/lossless-cut#1019] [Ideas]
Hi,I want to request the new feature,I had some gameplay footage in my youtube channel,but I need to cut out some part and merge to all in one file,and the missing feature is when I editing finished my footage in Batch file list,I need to export all file one by one,and merge again,I really need the feature can export one file after all batch list editing finished ,Thank you

**Comment by mifi** [comment]
Hi. I think your request will be coved by this issue https://github.com/mifi/lossless-cut/issues/89, please leave a thumbs up there
